tests/tasks/create_test_interfaces.yml

```diff
@@ -0,0 +1,42 @@
+# SPDX-License-Identifier: BSD-3-Clause
+---
+- name: Install dnsmasq
+  package:
+    name: dnsmasq
+    state: present
+
+
+- name: Create test interfaces
+  shell: |
+    # Manage all veth devices starting with test*
+    echo 'ENV{ID_NET_DRIVER}=="veth", ENV{INTERFACE}=="test*", ENV{NM_UNMANAGED}="0"' >/etc/udev/rules.d/88-veth.rules
+    udevadm control --reload-rules
+    udevadm control --reload-rules
```

Comment:
The other tests use veth as well but don't need this

Reply:
this is needed for NM to see test* device as managed directly after calling ip add, nevertheless removing this was missing and double udevadm control was incorrect too
    udevadm control --reload-rules
    udevadm settle --timeout=5